features/support/env.rb

```diff
@@ -22,28 +22,33 @@
 Capybara.app_host = ENV["GOVUK_WEBSITE_ROOT"]
 
 # Use Chrome in headless mode
+
+chrome_args = [
+  "headless",
+  "disable-gpu",
+  "disable-xss-auditor",
+  "--user-agent=Smokey\ Test\ \/\ Ruby"
+]
+chrome_args << ["no-sandbox"] if ENV.key?("NO_SANDBOX")
+
 Capybara.register_driver :headless_chrome do |app|
   capabilities = Selenium::WebDriver::Remote::Capabilities.chrome(
     acceptInsecureCerts: true,
+    'goog:chromeOptions': {
+      args: chrome_args
+    },
     'goog:loggingPrefs': {
       browser: "ALL",
       performance: "ALL"
     },
   )
 
-  options = Selenium::WebDriver::Chrome::Options.new
-  options.add_argument("--headless")
-  options.add_argument("--disable-gpu")
-  options.add_argument("--disable-xss-auditor")
-  options.add_argument("--user-agent=Smokey\ Test\ \/\ Ruby")
-  options.add_argument("--no-sandbox") if ENV.key?("NO_SANDBOX")
-
-  Capybara::Selenium::Driver.new(
-    app,
+  options = {
     browser: :chrome,
-    options: options,
-    desired_capabilities: capabilities
-  )
+    capabilities: capabilities
```

Comment:
```suggestion
    capabilities: [capabilities, options]
```

According to the deprecation message we can just combine the two:

> :options as a parameter for driver initialization is deprecated. Use :capabilities with an Array of value capabilities/options if necessary instead.

I've tested with an array and both sets are indeed passed forward.

Is there an advantage to the `chromeOptions` approach?